test/Datadog.Trace.ClrProfiler.IntegrationTests/AdoNet/MicrosoftDataSqlClientTests.cs

```diff
@@ -42,14 +42,15 @@ public void SubmitsTracesWithNetStandard(string packageVersion, bool enableCallT
             // Once this is fully supported, this will add another 2 complete groups for all frameworks instead
             // of 4 extra spans on net461 and netcoreapp2.0+
 #if NET461
-            var expectedSpanCount = 77; // 7 queries * 11 groups
+            var expectedSpanCount = 133; // 7 queries * 11 groups
 #else
-            var expectedSpanCount = 81; // 7 queries * 11 groups + 4 spans from generic wrapper on .NET Core
+            // Update to final value once tests pass
+            var expectedSpanCount = 137; // 7 queries * 11 groups + 4 spans from generic wrapper on .NET Core
```

Comment:
I guess this was the right number? Should probably update the comments

Reply:
I put off updating these 😅 Done in f9829db87629aa5c357cc4ad5a69a51fcd1da1ad